generate/templates/couchbase-edge-server/Dockerfile.template

```diff
@@ -0,0 +1,40 @@
+FROM {{ .DOCKER_BASE_IMAGE }}
+
+LABEL maintainer="docker@couchbase.com"
+
+ENV PATH $PATH:/opt/couchbase-edge-server/bin
+
+# Install dependencies:
+RUN set -x \
+    && apt update \
+    && apt install -y \
+           curl \
+           lsb-release \
+           systemctl \
+           wget \
+           zlib1g \
+    && apt clean
+
+RUN groupadd -g 1000 couchbase && useradd couchbase -u 1000 -g couchbase -M
```

Comment:
Just realized this isn't going to actually build, I don't think, because UID/GID 1000 already exist in `ubuntu:24.04` as the `ubuntu` user. That's a very annoying change since `ubuntu:22.04`. The following works instead:

    RUN groupmod --new-name couchbase ubuntu \
        && usermod --login couchbase -d / -g couchbase ubuntu \
        && rm -rf /home/ubuntu

Note this will *only* work if the `ubuntu` user already exists and is UID/GID 1000, so we may want to keep an eye on that in the future.

Reply:
apparently I have an older version of 22.04, didn't catch this.